lib/arg_types.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-console */
```

Comment:
I'm not too sure why there's a need for `console.log` in this file, but have ignored the rule for now.